docs/configure/policy/rules.md

```diff
@@ -55,6 +60,37 @@ if (ActionItem.Severity >= DANGER_SEVERITY && ActionItem.IsNew) {
 }
 ```
 
+```js
+if (ActionItem.Severity >= DANGER_SEVERITY && ActionItem.IsNew) {
+  sendSlackNotification(
```

Comment:
Also mind adding `sendSlackNotification` arguments information like how you did for `createTicket`?

Reply:
Good call!